src/components/Footer/Footer.styled.ts

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+import { Container } from 'components/primitives/Layout';
+import { Dot as GenericDot } from 'components/ActionsBuilder/AddEditOptionModal/AddEditOptionModal.styled';

```

Comment:
Should we move this to `components`?

Maybe into `Primitives` like the `Bullet`component

Reply:
Didn't knew about `Bullet` component. I am using `Bullet` now and also refactored `AddEditOptionModal` since `Dot` was almost the same as `Bullet`. Thanks